types/navermaps/index.d.ts

```diff
@@ -1234,12 +1235,12 @@ declare namespace naver.maps {
         class HeatMap {
             constructor(heatMapOptions?: HeatMapOptions);
             getColorMap(): SpectrumStyle;
-            getData(): LatLng[] | PointArrayLiteral[];
+            getData(): LatLng[] | PointArrayLiteral[] | WeightedLocation[];
```

Comment:
Only LatLng[] and PointArrayLiteral[] can be return type on this method.

<img width="778" alt="image" src="https://user-images.githubusercontent.com/16171447/180364767-c7577e79-0270-4b78-91b0-18f54ad054f4.png">